lib/simulator-xcode-6.js

```diff
@@ -462,8 +467,9 @@ class SimulatorXcode6 extends EventEmitter {
       args.push('-ConnectHardwareKeyboard', opts.connectHardwareKeyboard ? '1' : '0');
     }
 
-    if (_.isBoolean(opts.disablePasteboardAutomaticSync)) {
-      // Improve launching simulator performance
+    if (_.isBoolean(opts.disablePasteboardAutomaticSync) && opts.disablePasteboardAutomaticSync) {
+      // Improve launching simulator performance.
+      // Respect system wide preference if disablePasteboardAutomaticSync was false.
       // https://github.com/WebKit/webkit/blob/master/Tools/Scripts/webkitpy/xcode/simulated_device.py#L413
       args.push('-PasteboardAutomaticSync', '0');
```

Comment:
how about

```
args.push('-PasteboardAutomaticSync', `${+opts.disablePasteboardAutomaticSync}`)
```

Reply:
I considered the way, but I kept a case not to apply `PasteboardAutomaticSync` as launching argument since it could set in system-wide preference by `defaults write com.apple.iphonesimulator PasteboardAutomaticSync -bool false`.
Or `pasteboardAutomaticSync: 'on'`, `pasteboardAutomaticSync: 'off'`, `pasteboardAutomaticSync: 'system'`?
(Just came up with this idea)

'on': force it on
'off': force it off (current approach)
'system': does not set this flag

Reply:
true/false/(null|undefined) would serve the same approach